Manually seed PyTorch PRNG to avoid random test failures (#309)
desy-ml/cheetah: CHANGELOG.md

### 🐆 Other

> - CI is run with fixed seed for the PRNG, preventing random test failures (see #309) (@Hespe)

Reviewer: ```suggestion
- Test seeds for CI are fixed to prevent random test failures (see #309) (@Hespe)
```
